packages/flame/lib/src/components/mixins/collidable.dart

```diff
@@ -4,13 +4,12 @@ import '../../../components.dart';
 import '../../../game.dart';
 import '../../geometry/rectangle.dart';
 
-/// [CollidableType.active] collides with other [Collidable]s of type active or static
-/// [CollidableType.passive] collides with other [Collidable]s of type active
-/// [CollidableType.inactive] will not collide with any other [Collidable]s
+/// The [CollidableType] is used to determine which other type of [Collidable]s
+/// that your component should collide with.
 enum CollidableType {
-  active,
-  passive,
-  inactive,
+  active, // Collides with other [Collidable]s of type active or passive.
```

Comment:
use dartdocs instead?
eg
```
/// Collides with other [Collidable]s of type active or passive.
active,
```

Reply:
Oh, accidentally removed one `/`

Reply:
also the positioning? (can you have dartdocs after the name like that)?

Reply:
Nope, that's also fixed now